### ISSUE TITLE
Allow seeding with .graphql files containing multiple operations

### ISSUE THREAD

**Issue by marktani** [prisma/prisma#1867]
# Feature Request

**What feature are you missing?**

It's currently not possible to seed using a file like this:

```graphql
mutation a {
  createUser(data: { ... }) { id }
}

mutation b {
  createUser(data: { ... }) { id }
}
```

Instead, only this is possible:

```graphql
mutation a {
  a: createUser(data: { ... }) { id }

  b: createUser(data: { ... }) { id }
}
```

**How could this feature look like in detail? Tradeoffs?**

The first file above should be supported as seed file.

**Comment by roycclu** [comment]
Yes, need this feature for multiple seeding. What's the best solution or workaround at hand?

**Comment by marktani** [comment]
I mentioned a workaround above, in the second code snippet.

**Comment by divyenduz** [comment]
This is available in the latest version `1.25.0`, `npm install -g prisma`. I am closing this issue. Please let me know if the issue still persists via opening a new issue.
